server/util/Cargo.toml

```diff
@@ -9,7 +9,7 @@ path = "src/lib.rs"
 [dependencies]
 sha2 = "0.9.5"
 uuid = { version = "0.8", features = ["v4"] }
-chrono = "0.4.19"
```

Comment:
The workspace version has a `serde` feature but I don't think its an issue to use the workspace version here...